juniper_official/Solutions/Mpls-blackhole-detection/rsvpsession.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
403
/home/jenkins/.jenkins/workspace/iceberg/jfit-rules_validate/rules/juniper_official/Solutions/Mpls-blackhole-detection/rsvpsession.yml
  5:16      warning  truthy value is not quoted  (truthy)
  12:1      error    trailing spaces  (trailing-spaces)

Reply:
Addressed and resolved